deposit/bytecode-verification/run.sh

```diff
@@ -0,0 +1,105 @@
+  SPECS+=" "~/verified-smart-contracts/specs/deposit/get_deposit_count-spec.k
+  SPECS+=" "~/verified-smart-contracts/specs/deposit/deposit-init-spec.k
+  SPECS+=" "~/verified-smart-contracts/specs/deposit/deposit-subcall_1-spec.k
+  SPECS+=" "~/verified-smart-contracts/specs/deposit/deposit-subcall_2-spec.k
+  SPECS+=" "~/verified-smart-contracts/specs/deposit/deposit-log-spec.k
+  SPECS+=" "~/verified-smart-contracts/specs/deposit/deposit-data-spec.k
+  SPECS+=" "~/verified-smart-contracts/specs/deposit/deposit-add-init-then-spec.k
+  SPECS+=" "~/verified-smart-contracts/specs/deposit/deposit-add-init-else-spec.k
+  SPECS+=" "~/verified-smart-contracts/specs/deposit/deposit-add-loop-enter-then-spec.k
+  SPECS+=" "~/verified-smart-contracts/specs/deposit/deposit-add-loop-enter-else-spec.k
+  SPECS+=" "~/verified-smart-contracts/specs/deposit/deposit-add-loop-exit-spec.k
+  #
+  SPECS+=" "~/verified-smart-contracts/specs/deposit/deposit-init-calldata-spec.k
+  SPECS+=" "~/verified-smart-contracts/specs/deposit/deposit-init-calldata-revert-1-spec.k
+  SPECS+=" "~/verified-smart-contracts/specs/deposit/deposit-init-calldata-revert-2-spec.k
+  SPECS+=" "~/verified-smart-contracts/specs/deposit/deposit-init-calldata-revert-3-spec.k
+  SPECS+=" "~/verified-smart-contracts/specs/deposit/deposit-init-calldata-revert-4-spec.k
+  SPECS+=" "~/verified-smart-contracts/specs/deposit/deposit-init-calldata-revert-5-spec.k
+  SPECS+=" "~/verified-smart-contracts/specs/deposit/deposit-init-calldata-revert-6-spec.k
+  #
+  SPECS+=" "~/verified-smart-contracts/specs/deposit/deposit-init-revert-1-spec.k
+  SPECS+=" "~/verified-smart-contracts/specs/deposit/deposit-init-revert-2-spec.k
+  SPECS+=" "~/verified-smart-contracts/specs/deposit/deposit-data-revert-spec.k
+  #
+  SPECS+=" "~/verified-smart-contracts/specs/deposit/revert-invalid_function_identifier-lt_4-spec.k
+  SPECS+=" "~/verified-smart-contracts/specs/deposit/revert-invalid_function_identifier-ge_4-lt_32-spec.k
+  SPECS+=" "~/verified-smart-contracts/specs/deposit/revert-invalid_function_identifier-ge_4-ge_32-spec.k
+  SPECS+=" "~/verified-smart-contracts/specs/deposit/revert-init-spec.k
+  SPECS+=" "~/verified-smart-contracts/specs/deposit/revert-get_deposit_root-spec.k
+  SPECS+=" "~/verified-smart-contracts/specs/deposit/revert-get_deposit_count-spec.k
+
+  export LOGDIR=log.`date "+%F-%T-%Z" | sed 's/://g'`
+  mkdir $LOGDIR
+
+  bash lemmas.sh >lemmas.k
+
+  make clean
+  make split-proof-tests
+
+  run_kprove() {
+    kprove -v --debug \
+      -d ~/evm-semantics/.build/defn/java \
+      -m VERIFICATION $OPTS \
+      --smt-prelude ~/verified-smart-contracts/deposit/bytecode-verification/evm.smt2 \
+      --concrete-rules "EVM-DATA.take.zero-pad" \
+      `eval echo $1` \
+      >$LOGDIR/`basename $1`.log 2>&1
+    tail -100 $LOGDIR/`basename $1`.log
+  }
+  export -f run_kprove
+
+  echo $SPECS | xargs -d ' ' -n 1 -P 2 -I {} bash -c 'run_kprove "{}"'
+
+  date
```

Comment:
I guess the diff between this script and makefile is:
1. Extra command options.
2. Different set of specs to run.
3. the `tail` command.

I'd recommend moving all what is possible to Makefile, to keep minimalistic script structure. First 2 features can be implemented in makefile as extra options. The 3rd also can be done in Makefile as alternative command compared to `make test` or whatever is default. I can help if you want.